index.html

```diff
@@ -0,0 +1,322 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Mobile Website</title>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.sandbox.google.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="stylesheet" href="./styles.css" />
+  </head>
+  <body>
+    <div class="toolbar">
+      <nav class="nav-bar">
+        <a href="#"><img src="./images/logo.png" alt="logo" /></a>
+        <ul class="sid-cons">
+          <li class="p">Portoflio</li>
+          <li class="a">About</li>
+          <li class="c">Contact</li>
+          <li class="material-symbols-outlined">menu</li>
+        </ul>
+      </nav>
+    <div class="headline-container">
+      <div class="items1">
+        <h1 class="items item1">I'm Gasangwa Glad to see you !</h1>
+        <p class="items item3">
+          I am a software developer. I can help you build a product, feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need to be coded, don't hesitate to
+          contact me.
```

Comment:
- [ ] While working on the desktop version, You broke the mobile design. Please make sure the mobile version should still looks like the Figma design.

### See the images for reference 

![image](https://user-images.githubusercontent.com/83803414/166201927-dfd63637-8cb9-4b72-a37b-94a519c3a38d.png)

![image](https://user-images.githubusercontent.com/83803414/166201947-6ba412c3-2e59-4666-8f1c-0894a1f67352.png)

![image](https://user-images.githubusercontent.com/83803414/166201983-90192f7c-128c-447a-b155-14748ae380eb.png)

![image](https://user-images.githubusercontent.com/83803414/166202033-724d3a25-e4ad-43ae-9e5c-f1c8060fdfe4.png)